test/test_torch.py

```diff
@@ -1748,6 +1748,40 @@ def _test_neg(self, cast):
     def test_neg(self):
         self._test_neg(self, lambda t: t)
 
+    def test_bitwise_not(self):
+        res = 0xffff - torch.arange(127, dtype=torch.int8)
+        for t in (torch.BoolTensor,
```

Comment:
nit: can you, please, change variable names to something more readable? for example, 'dtype' instead of 't' and so on.

Reply:
Yes. I made the corresponding changes in #22320 and #22326 as those are the places where dtype is used.